manifests/charts/istiod-remote/files/injection-template.yaml

```diff
@@ -32,7 +32,7 @@ template: |
   {{- if excludeInboundPort (annotation .ObjectMeta `status.sidecar.istio.io/port` .Values.global.proxy.statusPort) (annotation .ObjectMeta `traffic.sidecar.istio.io/excludeInboundPorts` .Values.global.proxy.excludeInboundPorts) }}
     - "15090,15021,{{ excludeInboundPort (annotation .ObjectMeta `status.sidecar.istio.io/port` .Values.global.proxy.statusPort) (annotation .ObjectMeta `traffic.sidecar.istio.io/excludeInboundPorts` .Values.global.proxy.excludeInboundPorts) }}"
   {{- else }}
-  - "15090,15021"
+    - "15090,15021"
```

Comment:
Either old one was wrong and not tested or new one is wrong.
Is this the right spot?

Reply:
Old one was not tested because we do not update the golden files anymore. Although I think this is fixed in #25719 so not sure why it still shows up in the diff